tests/App/AppKernel.php

```diff
@@ -39,17 +39,22 @@ public function __construct()
 
     public function registerBundles()
     {
-        return [
+        $bundles = [
             new FrameworkBundle(),
             new TwigBundle(),
             new TwigExtraBundle(),
             new SecurityBundle(),
             new KnpMenuBundle(),
             new SonataBlockBundle(),
-            new SonataCoreBundle(),
             new SonataDoctrineBundle(),
             new SonataAdminBundle(),
         ];
+
+        if (class_exists(SonataCoreBundle::class)) {
```

Comment:
I think this can be avoided on all builds, just register both SonataTwig and SonataForm

Reply:
Thanks :+1:

Reply:
This will be work with `--prefer-lowest`. We can remove it when we up all sonata-project bundle where CoreBundle is optional. Not possible for now.